solid2/Glossary.md

```diff
@@ -0,0 +1,15 @@
+# Glossary
```

Comment:
Instead of duplicating the glossary, maybe we can link to the one under `/solid/Glossary.md`? Unless we need to add something specific related with the library, in that case I guess we could have a dedicated glossary as well but I'd reference the basic terms instead of duplicating them.